python/dask_cudf/dask_cudf/backends.py

```diff
@@ -498,17 +498,9 @@ def _unsupported_kwargs(old, new, kwargs):
         )
 
 
-def _raise_unsupported_parquet_kwargs(
-    open_file_options=None, filesystem=None, **kwargs
-):
+def _raise_unsupported_parquet_kwargs(filesystem=None, **kwargs):
     import fsspec
 
-    if open_file_options is not None:
-        raise ValueError(
-            "The open_file_options argument is no longer supported "
-            "by the 'cudf' backend."
-        )
-
```

Comment:
The failure you shared earlier was coming from cudf (`ValueError: cudf engine doesn't support the following keyword arguments: ['open_file_options']`) rather than dask-cudf, so I take it this check couldn't be causing the problem? Sorry - I'm a bit confused.

Reply:
The error is coming from [here](https://github.com/rapidsai/cudf/blob/branch-24.10/python/cudf/cudf/io/parquet.py#L936) in cudf, but that's being triggered via dask passing in kwargs that cudf doesn't support. You can see the failures in this PR's CI now, if that helps.

Reply:
>You can see the failures in this PR's CI now, if that helps.

I'll build your branch and reproduce locally. My intuition is that the failures will go away if you remove all changes from `python/dask_cudf/dask_cudf/io/parquet.py`, but there is a reasonable change I'm wrong (will follow up).

Reply:
OK, we've resolved this. Reverting all code changes to `dask_cudf` code in this PR was a fine solution since it already handled the argument. I just removed/modified tests as needed.